tests/framework/extensions/kubeapi/webhook/getwebhook.go

```diff
@@ -0,0 +1,34 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	clientV1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	V1 "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GetWebhook is a helper function that uses the dynamic client to get a list of webhooks from a cluster
+func GetWebhook(client *rancher.Client, clusterID, resourceName string) (*V1.ValidatingWebhookConfiguration, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(clusterID)
+	if err != nil {
+		return nil, err
+	}
+	webHookResource := schema.GroupVersionResource{
+		Group:    "admissionregistration.k8s.io",
+		Version:  "v1",
+		Resource: "validatingwebhookconfigurations",
+	}
```

Comment:
Could you check other kubeapi packages? Using this as a public variable might be beneficial, so if we need it in the future, we can use it.

Reply:
Resolved in the other PR. Once that is merged will rebase this PR again with those changes.